packages/app/test/cypress/integration/21-basic-features-for-guest/access-to-page.spec.ts

```diff
@@ -8,6 +8,12 @@ context('Access to page by guest', () => {
 
   it('/Sandbox is successfully loaded', () => {
     cy.visit('/Sandbox', {  });
+    // Collapse sidebar
+    cy.getByTestid('grw-contextual-navigation-sub').then(($el) => {
+      if(!$el.hasClass('d-none')){
+        cy.getByTestid('grw-navigation-resize-button').click({force: true});
+      }
+    });
```

Comment:
`cy.collapseSidebar(true);` doesn't work here as well?

Reply:
I have tried using `cy.collapseSidebar(true)` before access-to-page-by-guest--sandbox screenshot but not working. 
It also doesn't work on `beforeEach`

Result
https://growi-vrt-snapshots.s3.amazonaws.com/b189b8ef458d9365f352f58b5481ec26f0e055b1/index.html?id=passed-21-basic-features-for-guest/access-to-page.spec.ts/access-to-page-by-guest--sandbox.png#passed-21-basic-features-for-guest/access-to-page.spec.ts/access-to-page-by-guest--sandbox.png

Reply:
Could you check why it is not working?

I'm wondering if : 
- the code in `packages/app/test/cypress/support/commands.ts` needs fixing?
- the element or `data-testid` specified in the codes does not exist?
- cy.wait is too short?